test/acceptance/dummy_test.exs

```diff
@@ -0,0 +1,9 @@
+defmodule PRM.DummyAcceptanceTest do
+  use PRM.Support.AcceptanceCase
+
+  test "api" do
+    "/api"
+    |> get!()
+    |> assert_status(404)
```

Comment:
@PavelVesnin can we assert on an API that returns 200? I think `/api/doctors` should do exactly that.